dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/notification/InMemoryNotifier.java

```diff
@@ -122,7 +123,7 @@ public Notifier update( JobConfiguration id, NotificationLevel level, String mes
     }
 
     @Override
-    public Map<JobType, Map<String, LinkedList<Notification>>> getNotifications( )
+    public Map<JobType, LinkedHashMap<String, LinkedList<Notification>>> getNotifications( )
```

Comment:
Specific reason for not using Map and List interfaces here (usually better) ?

Reply:
With ordering it is easier to remove the oldest entry